draft-ietf-taps-impl.md

```diff
@@ -958,7 +958,7 @@ Initiate:
 : CONNECT.UDP. Calling `Initiate` on a UDP Connection causes it to reserve a local port, but does not generate any traffic.
 
 InitiateWithSend:
-: Early data on a UDP Connection does not have any special meaning. The data is sent whenever the Connection is Ready.
+: Early data on a UDP Connection does not have any special meaning. The data is sent whenever the Connection is `Ready`.
```

Comment:
```suggestion
: Early data on a Connection that uses UDP does not have any special meaning. The data is sent whenever the Connection is `Ready`.
```

Reply:
Not sure I agree, since we define above:
```
Connection Object:
: UDP Connections represent a pair of specific IP addresses and ports on two hosts.
```

I think that is not in scope for this PR...